examples/gallery/symbols/custom_symbols.py

```diff
@@ -0,0 +1,49 @@
+"""
+Custom symbols
+--------------
+
+The :meth:`pygmt.Figure.plot` method can plot individual custom symbols
+by passing the corresponding symbol name together with the **k** shortcut to
+the ``style`` parameter. In total 41 custom symbols are already included of
+which the following plot shows five examplary ones. The symbols are shown
+underneath their corresponding names. For the remaining symbols see the GMT
+cookbook :gmt-docs:`cookbook/custom-symbols.html`.
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+fig.basemap(region=[0, 8, 0, 3], projection="X12c/4c", frame=True)
+
+# define pen and fontstlye for annotations
+pen = "1p,black"
+font = "15p,Helvetica-Bold"
+
+y = 1.25
```

Comment:
Shall we use the number in `fig.plot` and `fig.text` directly since we only plot five symbols, like another gallery example for [multi-parameter symbols](https://pygmt-l6f9m2d1j-gmt.vercel.app/gallery/symbols/multi_parameter_symbols.html#sphx-glr-gallery-symbols-multi-parameter-symbols-py)?